src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentForm.java

```diff
@@ -37,7 +39,8 @@ public IWantDocumentForm() {
         newSourceLine = new IWantAccount();
         this.setDeptOrgKeyLabels(new ArrayList<KeyValue>());
         editingMode = new HashMap<Object,Object>();
-
+        presentationMode = CUPurapConstants.IWantPresentationModes.MULTIPLE_PAGE_MODE;
+        usersRequestingPresentationModeChange = true;
```

Comment:
I'm only seeing `usersRequestingPresentationModeChange` being set to true. Is it supposed to lose its value and revert to false if the user takes a document action that doesn't call the related setter? If so, could the variable be renamed to make its intent clearer, or could this be implemented in a clearer fashion?

Reply:
Thanks for asking about this boolean flag. It was used during initial development of this feature but subsequent logic adjustments have resulted in it no longer being needed. I have removed it.